lib/src/client/call.dart

```diff
@@ -179,7 +197,11 @@ class ClientCall<Q, R> implements Response {
   }
 
   void _onTimedOut() {
-    _responses.addError(GrpcError.deadlineExceeded('Deadline exceeded'));
+    final error = GrpcError.deadlineExceeded('Deadline exceeded');
+    _timeline?.finish(arguments: {
+      'timeout': error.toString(),
```

Comment:
This should probably be using `error` as the key.

Reply:
Done